examples/lang2/delegator/build-all.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env
```
Since depending on the OS bash is not always in `/bin` and `#!/usr/bin/env` searches the `PATH` I suggest going for this version.

Normally I would also suggest to add something like `set -euo pipefail`. I think it's good practice to have this as a preamble in all shell scripts. In this particular script it doesn't have any benefit though.

Reply:
Thanks!